src/app/taquin/taquin.component.css

```
@@ -0,0 +1,113 @@
+.centre{
+	margin: auto;
+	/* text-align: center; */
+}
+
+.main9{
+	margin: auto;
+	text-align: center;
+	border: solid black 2px;
+	width: 310px;
+	height: 310px;
+	display: grid;
+	grid-template-columns: 1fr 1fr 1fr;
+	grid-template-rows: 1fr 1fr 1fr;
+	border-radius: 3px;
+}
+
+.main16{
+	margin: auto;
+	text-align: center;
+	border: solid black 2px;
+	width: 390px;
+	height: 370px;
+	display: grid;
+	grid-template-columns: 1fr 1fr 1fr 1fr;
+	grid-template-rows: 1fr 1fr 1fr 1fr;
+}
+
+.mainWin9{
```

Prefer `kebab-case` for class names
```suggestion
.main-win-9{
```